integration-tests/constants/zendeskFormFieldIDs.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+
+export enum ZendeskFormFieldIDs {
```

Comment:
What are we gaining from making this an enum in a separate file? Can we move this back to the previous file?

If this was done for organisation and clarity, we can make the existing `zendeskParameters.ts` file a bit clearer with simple key value objects.

For example:
```
export const PII_FORM_FIELD_IDS = {
  IDENTIFIER: 5605352623260
  EVENT_ID: 5605423021084
  DATE: 5605700069916
  REQUESTED_PII_TYPE: 5641719421852
  CUSTOM_DATA_PATH: 5698447116060
  REQUEST_STATUS_FIELD_ID: 5605885870748
  RECIPIENT_EMAIL: 6202354485660
  RECEIPIENT_NAME: 6202301182364
}
```

Note: I left out `ZENDESK_PII_FORM_ID = 5603412248860` because this is not a field id, it's a form id.

You could also do this for the other grouping.

Reply:
You would then reference them the same way e.g. `PII_FORM_FIELD_IDS.IDENTIFIER`

Reply:
I did that for better organisation. I didn't know enums could live in the same file as other things. Done.

Reply:
Can we not make this an enum though? An object with key values would be much better. Means we don't need to disable linting rules which you seem to have done.